brave/browser-laptop
# Pending/confirmed sync records to allow pause/resume

~~Depends on https://github.com/brave/sync/pull/122~~ *merged*

Fix https://github.com/brave/sync/issues/125

Submitter Checklist:


Test Plan:
1. From Pyramid 0 prepare 1000s of bookmarks and setup Sync.
2. Examine sync network activity with the Sync extension page: chrome-extension://cjnmeadmgmiihncdidmfiabhenbggfjm/_generated_background_page.html
3. Close the browser while sync is in progress.
4. Open the browser again and allow sync to finish. Eventually sync completes when there's no more network activity.
5. Setup Pyramid 1 on the Sync profile and wait for it to download records. Confirm that all bookmarks from Pyramid 0 are present.


Reviewer Checklist:

Tests
